src/app/app.component.ts

```diff
@@ -25,11 +25,7 @@ import { filter } from "rxjs/operators";
  */
 @Component({
   selector: "app-root",
-  template: `@if (configFullscreen) {
-      <router-outlet></router-outlet>
-    } @else {
-      <app-ui></app-ui>
-    }`,
```

Comment:
For this, I have created the `app.component.html` file and used it directly here.

Reply:
Is this directly related to your PR?

If not, let's not do it as part of this branch :blush:
Separate template file may be preferable and this is a simple change but having unrelated code has sometimes bloated PRs or even led to side-effects previously, so let's be strict with "one task, one PR"